test/pagespecs/DebtTooLargePageSpec.scala

```diff
@@ -24,7 +24,7 @@ class DebtTooLargePageSpec extends ItSpec {
   "language" in {
 
     debtTooLargePage.open()
-    debtTooLargePage.assertPageIsDisplayed()
+    debtTooLargePage.assertPageIsDisplayed
```

Comment:
Actually I did the parentheses intentionally. 

Methods with empty parameter lists are, by convention, evaluated for their side-effects. Methods without parameters are assumed to be side-effect free. That's the convention.

Reply:
I won't block it from merging

Reply:
Sorry that was unintentional. While I was building out the specs I parametrised that call for a while then decided that was a bad approach so undid it. I must have removed the parameters along with the arg I was passing in. Will put them back when I touch this again.